lib/cocoapods/executable.rb

```diff
@@ -53,7 +53,7 @@ def self.execute_command(executable, command, raise_on_failure)
 
       require 'open4'
 
-      full_command = "#{bin} #{command}"
+      full_command = "\"#{bin}\" #{command}"
```

Comment:
What if the bin has a `"` in the name, this will break still. Instead you should shell escape it:

```bash
$ ln -s /usr/bin/false "fal\" se"
$ ls | grep fal
fal" se
```

### Currently

```ruby
require 'open4'

false_command = 'fal" se'
Open4.spawn("\"#{false_command}\"")

Open4::SpawnError: cmd <"fal" se"> failed with status <2> signals <>
from /Users/kyle/gems/gems/open4-1.3.4/lib/open4.rb:366:in `spawn'
```

### Shellescape

```ruby
require 'shellwords'

false_command = 'fal" se'
Open4.spawn(false_command.shellescape)

Open4::SpawnError: cmd <fal\"\ se> failed with status <127> signals <>
from /Users/kyle/gems/gems/open4-1.3.4/lib/open4.rb:366:in `spawn'
```